index.html

```diff
@@ -54,7 +58,123 @@
       YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
 
     -->
+<header>
+    <h2> Ottawa Electronics</h2>
+    <img src="lightning-logo.png" alt="ottawa electronics logo">
+<nav>
+  <h3>main site menu</h3>
+  <ul>
+    <lI><a href="#">Home Page</a></lI>
+    <li><a href="#">About us</a></li>
+    <li><a href="#">Contact us</a></li>
+  </ul>
+</nav>
+</header>
+ <main>
 
-    <h1>Assignment 01 - To Be Completed</h1>
+  <section>
+  <h1>Ottawa's number 1 Electronics retailer</h1>
+  <img src="ottawa lightening.jpeg" alt="lightening strikes downtown Ottawa">
+  </section>
+
+  <Section>
+<h2>costumer testimonials</h2>
+  <p>"This store has the best  costumer service of all time.
+  the moment I walked in the store, was the single most joyus moment of my existence.11/10"
+  
+  -Jane Doe
+  </p>
+
+  <P>
+    "very reasonable pricing, my son loves his brand new gamestation 5. 8/10"  
+    -Bob Stan
+  </P>
+
+  <p>
+    "wonderful stores, I truly love the wild variety of products. 9/10"
+    -Tom Ryan
+  </p>
+</section>
+
+<section>
+  <h2> Products</h2>
+  <H3>Gamestation 5</H3>
+<img src="GS5.webp" alt="Gamestation 5">
+
+<P>
+  Gamestation 5 is an all new hightech gaming console. for the price of only $5,000, you can reach the pinnacle of gaming in your living room.
```

Comment:
All headings should be wrapped around a section tag. 

Have a look at the course notes for how to do sections and headings correctly